### PR TITLE
Add the Security Policy Link from the Contributor Guide

### LINKED ISSUE #20153: Missing Security Policy details from the Contributor Guide
**Description:**
The contributor guide [1] is linked from ballerina community page [2]. In the Contributor guide, under the section "Filing Issues" it says "See the security policy for details.", but the link is missing.

image

[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md
[2] https://ballerina.io/community/

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Remove the "security policy" link in [1] as it might not be relevant.

[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md

Fixes #20153 

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by anupama-pathirage** [comment]
@praneesha Although the topic and description says this is to remove the link, it actually has included the link right?

**Comment by praneesha** [comment]
@anupama-pathirage 

> @praneesha Although the topic and description says this is to remove the link, it actually has included a link right?

Changed the title of the PR to "Add the Security Policy Link from the Contributor Guide".

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ We appreciate your help!
 
 If you are unsure whether you have found a bug, please consider searching existing issues in github and asking in Ballerina-Dev Google Group.
 
-> IMPORTANT: Sensitive security-related issues should be reported to [security@ballerina.io](security@ballerina.io). See the security policy for details.
+> IMPORTANT: Sensitive security-related issues should be reported to [security@ballerina.io](security@ballerina.io). For details, see the [security policy](https://ballerina.io/security/)].
```
Comment (anupama-pathirage) [inline CONTRIBUTING.md]: IMO, we should include the same link from https://github.com/ballerina-platform/ballerina-dev-website/blob/master/community.md under "Want to Report Issues" too.

Comment (praneesha) [reply CONTRIBUTING.md]: Added this via https://github.com/ballerina-platform/ballerina-dev-website/pull/131/commits/478371da91238165498cb79c72d29b75cfd2cb02.

Comment (anupama-pathirage) [inline CONTRIBUTING.md]:
```suggestion
> IMPORTANT: Sensitive security-related issues should be reported to [security@ballerina.io](security@ballerina.io). For details, see the [security policy](https://ballerina.io/security/).
```

**Pull request merged**